javalib/src/main/scala/java/util/function/BiFunction.scala

```diff
@@ -0,0 +1,22 @@
+/*
+ * Scala.js (https://www.scala-js.org/)
+ *
+ * Copyright EPFL.
+ *
+ * Licensed under Apache License 2.0
+ * (https://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+
+package java.util.function
+
+import scala.scalajs.js.annotation.JavaDefaultMethod
+
+trait BiFunction[T, U, R] { self =>
```

Comment:
This commit needs to go before the one adding `BinaryOperator` because `BinaryOperator` depends on `BiFunction`